infrastructure/general/setup-bootnode.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set up a Boot Node
+description: Instructions on setting up, testing, maintaining, and testing a boot node.
+---
+
+!!!note
```

Comment:
This shouldn't be a note. Add an ##Introduction header and explain a little bit more in there,  and link somewhere else if needed